client/src/base-components/Drawer/Drawer.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { MUI_PLACEMENTS } from '@utils';
+import {
+  Box,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  MuiDrawer,
+} from './Drawer.styled';
+
+export default function Drawer({
+  anchor,
+  children,
+  childrenListStyle,
+  itemsList,
+  itemsListStyle,
+  onClose,
+  onOpen,
+  open,
+  variant,
+  ...props
+}) {
+  return (
+    <MuiDrawer
+      anchor={anchor}
+      childrenListStyle={childrenListStyle}
+      itemsListStyle={itemsListStyle}
+      onClose={onClose}
+      onOpen={onOpen}
+      open={open}
+      variant={variant}
+      {...props}
+    >
+      <List>
```

Comment:
make sure you have at lease one item to insert this List tag here 
`{itemsList?.length ? (<List> . . . </List>) : null}`